src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<blockquote>knowledge in Python and the <strong>librairies </strong>that can be used</blockquote><p>--&gt; libraries</p><p> </p><blockquote>(we only used 2 : numpy for easier data storage and mathematic operations as well as pyplot for the graph)</blockquote><p>(simple reminder) If you accept my suggestion about unit conversion, that would make 3 of them.</p><p> </p><blockquote>but the construction would be manual and would need to be <strong>change </strong>for each problem.</blockquote><p>--&gt; changed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Changed in 4d319923